tardis/tests/test_tardis_full.py

```diff
@@ -74,24 +71,24 @@ def get_ref_data(key):
 
         return get_ref_data
 
-    def test_j_blue_estimators(self, transport, refdata):
-        j_blue_estimator = refdata("j_blue_estimator").values
+    # def test_j_blue_estimators(self, transport, refdata):
```

Comment:
I assume these comments are temporary

Reply:
yes, should be fixed once I make the RadiationFieldMCEstimators a subclass of `HDFWriterMixin` to save it as an HDF.